### PR TITLE
Add templating

### PR DESCRIPTION
**Pull request by Tinitine**

### CODE REVIEW

File: src/lib/server/pdf/index.ts [hunk 1]
```diff
@@ -53,19 +51,29 @@ const fonts: TFontDictionary = {
...
+	ArialNarrow:{
+		normal: 'src/lib/assets/fonts/arial-narrow.ttf',
+		bold: 'src/lib/assets/fonts/arial-narrow-bold.ttf',
+		italics: 'src/lib/assets/fonts/arial-narrow-italic.ttf'
 	}
 };
 
-// format of the pdf  
 
+type Alignment = 'center' | 'right'
```
Comment (pitzzahh) [inline src/lib/server/pdf/index.ts:63]: Move types to type.ts
@Paotatoo
@Tinitine

File: src/lib/assets/icons/mail.png
Comment (pitzzahh) [inline src/lib/assets/icons/mail.png:1]: Add padding sa mail icon kasi dikit masyado sa edges 
@Tinitine

File: src/lib/server/pdf/index.ts [hunk 2]
```diff
@@ -53,19 +51,29 @@ const fonts: TFontDictionary = {
...
+type TableData = {
```
Comment (pitzzahh) [inline src/lib/server/pdf/index.ts:64]: Also mote to types.ts
@Paotatoo

File: src/lib/server/pdf/index.ts [hunk 3]
```diff
@@ -53,19 +51,29 @@ const fonts: TFontDictionary = {
...
+	text: string,
+	alignment: Alignment
+}
 
 const printer = new PdfPrinter(fonts);
 
 export const generatePDF = async (
```
Comment (pitzzahh) [inline src/lib/server/pdf/index.ts:71]: Add another prop for the signing officials.

File: src/lib/server/pdf/index.ts (hunk 1 above)
Comment (pitzzahh) [inline src/lib/server/pdf/index.ts:63]: Also move to type.ts

File: src/routes/api/pdf/[record]/+server.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { generatePDF } from '$lib/server/pdf';
-import type { EmployeeInfo, RecordAppointment, Service, ServiceRecord } from '$lib/types';
+import type { EmployeeInfo, RecordAppointment, Service, ServiceRecord, VerifiedBy} from '$lib/types';
 import type { TDocumentInformation } from 'pdfmake/interfaces';
 
 export const GET = async ({ params }) => {
```
Comment (pitzzahh) [inline src/routes/api/pdf/[record]/+server.ts:5]: should be a POST request

[Guide](https://learn.svelte.dev/tutorial/post-handlers)

File: src/lib/server/pdf/index.ts (hunk 1 above)
Comment (pitzzahh) [reply src/lib/server/pdf/index.ts:63]: dipa na move pati su iba

File: src/routes/api/pdf/[record]/+server.ts [hunk 5]
```diff
@@ -30,9 +30,13 @@ export const GET = async ({ params }) => {
...
 		salary: 122222
 	}
+	const veriBy: VerifiedBy = {
+		HRMOII: 'afvdf',
+		admin: 'adfgsg'
+	}
 	
 //should not be duplicate
-	const pdf = await generatePDF(doc,info, metadata);
+	const pdf = await generatePDF(doc,info,veriBy, [] ,metadata);
```
Comment (pitzzahh) [inline src/routes/api/pdf/[record]/+server.ts:39]: Add sample data

**Review by pitzzahh — CHANGES_REQUESTED** [review]

File: src/lib/assets/icons/mail.png
Comment (Tinitine) [reply src/lib/assets/icons/mail.png:1]: Okay po boss

**Review by pitzzahh — APPROVED** [review]

**Pull request merged**